### DISCUSSION TITLE
add a auction house to shattrath

### DISCUSSION THREAD

**Discussion by pablitoF** [chromiecraft/chromiecraft#4582] [Suggestions]
It is tedious to constantly move from old world to outland, It would be a quality of life improvement in my opinion to add a auctioneer to the Scryer and Aldor bank.

**Comment by Cooties-cc** [comment]
Just make a bank alt? I mean dal in wrath has one, but it's eng only. I wouldn't really agree that when it comes, it should be for everyone. I don't think it's that big of an issue to have one in shat when we do get ports back. Just AH when hearth is on CD, or mail to bank alt.
